docs/fiddles/features/macos-dock-menu/preload.js

```diff
@@ -0,0 +1,8 @@
+const { contextBridge } = require('electron');
```

Comment:
If exposing these APIs through `contextBridge` is unnecessary for this demo, maybe we should omit the preload entirely?

Reply:
I think we're going to hit this on every doc update - removing `nodeIntegration: true` breaks the "standard" fiddle home page.

But it looks like fiddle by default now uses:

```js
// All of the Node.js APIs are available in the preload process.
// It has the same sandbox as a Chrome extension.
window.addEventListener('DOMContentLoaded', () => {
  const replaceText = (selector, text) => {
    const element = document.getElementById(selector)
    if (element) element.innerText = text
  }

  for (const type of ['chrome', 'node', 'electron']) {
    replaceText(`${type}-version`, process.versions[type])
  }
})
```

maybe we should do that?

Reply:
Just went ahead and removed the versions.